### PR TITLE
support conv transpose in script

### PR DESCRIPTION
**Pull request by eellison**

### CODE REVIEW

File: torch/nn/modules/conv.py
Comment (wanchaol) [inline torch/nn/modules/conv.py]: nit: comment?

File: torch/nn/modules/conv.py
Comment (wanchaol) [inline torch/nn/modules/conv.py]: is the function signature change necessary?

**Review by wanchaol — APPROVED** [review]
LGTM if CI pass

File: torch/nn/modules/conv.py
Comment (eellison) [reply torch/nn/modules/conv.py]: Yep this needs a fix as well

File: torch/nn/modules/conv.py
Comment (eellison) [reply torch/nn/modules/conv.py]: The alternative is to make lists a constant of the module. By changing the signature, the tuples get converted to lists upon entry of the function, and we can index into the lists with a non-constant value

**Comment by zou3519** [comment]
Test failures look real, please don't land this

**Pull request closed without merging**
